javascript/javascript1/week2/student-manager.js

```diff
@@ -34,11 +34,7 @@ let istudentExist = studentName => class07Students.find(name => name.toLowerCase
 //     else {
 //         return true;
 //     }
-// }
-
-const getNoOfStudents = class07Students => console.log(`The number of total students in the class is  ${class07Students.length}`);
-
-getNoOfStudents(class07Students);
+//}
```

Comment:
.find() methode returns true or false by default , so you don't need to use ternary operator.

Reply:
👍🏻 got it,thank you.